tests/returntypes/test_qcut_new.py

```diff
@@ -0,0 +1,4753 @@
+# Copyright 2022 Xanadu Quantum Technologies Inc.
```

Comment:
This entire test file was copied from `tests/transforms/test_qcut.py`. It's completely unmodified, with the exception of added `xfail` markers to the torch tests, since the torch interface has not been implemented yet.